source/concepts/sdlc/user-workflow.html.md.erb

```diff
@@ -50,14 +50,15 @@ The code and AWS account is protected in a few different ways which work togethe
 #### CODEOWNERS
 
 Your GitHub team will be assigned as a [codeowner](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/about-code-owners) for your application folder, so someone in your team or the Modernisation Platform team will be required to review any pull requests before they can be merged.
+Your Github team will be able to approve a majority of pull requests. Approvals from the modernisation platform team members are only required in cases where a change might impact other customers or core platform components such as any files in the `.github` folder, as well as `providers.tf`, `backend.tf` and `networking.auto.tfvars.json` files in you application directory.
```

Comment:
`in **your** application directory`
Maybe instead of referring the the folders, inform them that they can look at the codeowners file?

Reply:
I have added a reference to the codeowners file.